lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -36,6 +38,8 @@ tokio = { version = "1.37", features = ["full"] }
 tokio-boring = { git = "https://github.com/gravitational/boring", rev="605253d99d5e363e178bcf97e1d4622e33844cd5", optional = true }
 utf16string = "0.2.0"
 uuid = { version = "1.8.0", features = ["v4"] }
+tracing-subscriber = "0.3.17"
```

Comment:
Are these new crates necessary?

Reply:
Hmm, I don't think so. @probakowski it looks like you may have added these before I took over for you, do we need to keep them for some reason?